CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Performance:
 
 
 Changes:
+* Introduce a new constructor to `RubyRange` & override methods which explicitly manipulate a `frozen` field on the object (#2570, @MattAlp)
```

Comment:
The changelog is used to communicate to end users what has changed, so we don't note internal refactorings that aren't user visible unless it fixes a bug or improves performance. If that's the case here, the changelog entry should highlight that. Otherwise, I think you back this out and update the changelog once you finish whatever larger thing you were working on.

Reply:
Thanks for the heads up, I'll nix it